.travis.yml

```diff
@@ -19,6 +19,8 @@ os:
   - osx
 script:
   - npm test
-
+after_success:
+  - npm run test:coverage
+  - cat ./coverage/lcov.info | ./node_modules/codecov/bin/codecov
```

Comment:
legit point. Haven't considered that since I assumed this'll be pretty much CI specific on *nix machines. Will give it a try and update PR shortly.